mlflow/tensorflow/reuters_classifier/README.rst

```diff
@@ -0,0 +1,113 @@
+Reuters classification for ODAHU
+====================================
+
+What's problem the model solve?
+-----------------------------------------
+
+The example shows how to solve document classification problem using ODAHU.
+
+Document classification or document categorization is a problem in library science, information science and computer science.
+The task is to assign a document to one or more classes or categories
+
+Well-known  `Reuters-21578 dataset <https://kdd.ics.uci.edu/databases/reuters21578/reuters21578.html>`_ is used for training the model.
+
+The purpose of this example is not to show best approach to solve the document classification problem but rather
+to show features of MLFlow library.
+
+`Keras Deep learning library <https://keras.io/>`_ is used to train model.
+
+Structure of example is shown below
+
+.. code-block:: bash
+
+   tree reuters_classifier
+
+::
+
+    reuters_classifier/
+    ├── MLproject
+    ├── README.rst
+    ├── conda.yaml
+    ├── odahuflow
+    │   ├── deployment.odahuflow.yaml
+    │   ├── git_connection.odahuflow.yaml
+    │   ├── packaging.odahuflow.yaml
+    │   └── training.odahuflow.yaml
+    ├── src
+    │   ├── parser.py
+    │   └── utils.py
+    └── train.py
+
+
+
+The example contains the following files:
+
+ * MLproject
+   Contains definition of this project. Contains only one entry point to train the model.
+
+ * conda.yaml
+   Defines project dependencies.
+
+ * train.py
+   Main entry point of the projects.
+
+ * odahuflow
+   directory with manifests to interact with ODAHU cluster
+
+ * src/parser.py
+   Contain classes to download and parse reuters dataset
+
+ * src/utils.py
+   Contain set of help functions and :ref:`ModelWrapper` class, that allow to add extra functionality to model
+
+
+Because of in general deep learning libraries require matrix of number features we should tokenize text corpus,
+and encode labels that try to predict.
```

Comment:
if I rightly understand.
```suggestion
Because of, in general, deep learning libraries require matrix of number features, we should tokenize text corpus, and encode labels that try to predict.
```